common/src/main/java/com/revenuecat/purchases/common/FileHelper.kt

```diff
@@ -0,0 +1,65 @@
+package com.revenuecat.purchases.common
+
+import android.content.Context
+import java.io.BufferedReader
+import java.io.File
+import java.io.FileInputStream
+import java.io.FileOutputStream
+import java.io.InputStreamReader
+
+class FileHelper(
+    private val applicationContext: Context,
+) {
+    fun appendToFile(filePath: String, contentToAppend: String) {
+        val file = getFileInFilesDir(filePath)
+        file.parentFile?.mkdirs()
+        val shouldAppend = true
+        val outputStream = FileOutputStream(file, shouldAppend)
+        outputStream.use {
+            outputStream.write(contentToAppend.toByteArray())
+        }
+    }
+
+    fun deleteFile(filePath: String): Boolean {
+        val file = getFileInFilesDir(filePath)
+        return file.delete()
+    }
+
+    fun readFilePerLines(filePath: String): List<String> {
+        val readLines = mutableListOf<String>()
+        val file = getFileInFilesDir(filePath)
+        FileInputStream(file).use { fileInputStream ->
+            InputStreamReader(fileInputStream).use { inputStreamReader ->
+                BufferedReader(inputStreamReader).use { bufferedReader ->
+                    readLines.addAll(bufferedReader.readLines())
+                }
+            }
+        }
+        return readLines
+    }
+
+    fun removeFirstLinesFromFile(filePath: String, numberOfLinesToRemove: Int) {
+        val readLines = readFilePerLines(filePath)
+        deleteFile(filePath)
+        val textToAppend = if (readLines.isEmpty() || numberOfLinesToRemove >= readLines.size) {
+            errorLog("Trying to remove $numberOfLinesToRemove from file with ${readLines.size} lines.")
+            ""
+        } else {
+            readLines.subList(numberOfLinesToRemove, readLines.size).joinToString(separator = "")
+        }
+        appendToFile(filePath, textToAppend)
+    }
+
+    fun fileIsEmpty(filePath: String): Boolean {
```

Comment:
Might be useful to document that this also returns `true` if the file doesn't exist.

Reply:
Sorry, merged this without pushing this. Added it to the `diagnostics` branch though: https://github.com/RevenueCat/purchases-android/commit/74d6c216e1214fdb8b70af26d33a592fc8b8cfb5